native-oci-generative-ai/README.md

```diff
@@ -0,0 +1,56 @@
+# OCI Native Generative AI Example
```

Comment:
Please add a GitHub workflow so that this demo is tested.

Reply:
Right. We could do this via [secrets](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions), but in this case, I would be ok with just building the app with Native Image.